scripts/scf_install.sh

```diff
@@ -5,6 +5,9 @@ set -ex
 . scripts/include/common.sh
 . .envrc
 
+# save CHART_URL on cap-values configmap
+kubectl patch -n kube-system configmap cap-values -p $'data:\n chart: "$CHART_URL"'
```

Comment:
Probably the same should happen during upgrade

Reply:
done in https://github.com/SUSE/bkindwscf/pull/25